addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -743,3 +743,21 @@ msgstr ""
 msgctxt "#31163"
 msgid "Show Fanart background"
 msgstr ""
+
+#. Label of a setting that, if enabled, shows the name of the currently logged-in profile in the UI
+#: /xml/SkinSettings.xml
+msgctxt "#31164"
+msgid "Choose kind of profile identification"
+msgstr ""
+
+#. Label for the kind of profile identification
+#: /xml/SkinSettings.xml
+msgctxt "#31165"
+msgid "Profile name"
```

Comment:
Name

Reply:
If you look at other labels (even those with 2 words) every other label has the 2nd word written in lower case. 

So, tbh not too sure if we should change that 'style'.

Reply:
Keep it as it is now.

Reply:
Yes for the settings labels we only ever have the first letter of the first word as a capital, everything other than the first word always begins in lowercase.